openelevationservice/server/api/elevation_query_parallel.py

```diff
@@ -0,0 +1,152 @@
+from sqlalchemy import text
+import json
+from shapely.geometry import shape, mapping, Polygon, MultiPolygon
+from shapely.ops import unary_union
+from multiprocessing import Pool
+import math
+
+#Start-Query code
+query = text(
+    """
+    WITH query_geom AS (
+    SELECT ST_SetSRID(ST_GeomFromText(:polygon), 4326) AS geom
+),
+polygons AS (
+    SELECT *
+    FROM (
+        SELECT 
+            (ST_PixelAsPolygons(
+                ST_Clip(oes_cgiar.rast, query_geom.geom, 0),
+                1, False
+            )).geom AS geometry,  -- Extrae las celdas como geometría
+            (ST_PixelAsPolygons(
+                ST_Clip(oes_cgiar.rast, query_geom.geom, 0),
+                1, False
+            )).val AS height  -- Extrae el valor de elevación
+        FROM query_geom 
+        JOIN oes_cgiar 
+        ON ST_Intersects(oes_cgiar.rast, query_geom.geom)
+    ) AS subquery
+    WHERE height != 0  
+)
+SELECT jsonb_build_object(
+    'type', 'FeatureCollection',
+    'features', jsonb_agg(jsonb_build_object(
+        'type', 'Feature',
+        'geometry', ST_AsGeoJSON(geometry),
+        'properties', json_build_object(
+            'heightBase', height
+        )
+    ))
+) AS features_collection,
+MIN(height) AS min_height,
+MAX(height) AS max_height,
+AVG(height) AS avg_height
+FROM polygons;
+
+    """
+)
+###End-Query code 
+
+
+####Start-Functions used in parallelization
```

Comment:
It is not necessary to comment on where a function starts and where it ends.

Reply:
Done. Deleted.